imap/caldav_alarm.c

```diff
@@ -1702,7 +1697,7 @@ static int process_snoozed(struct caldav_alarm_data *data,
 
     /* Check runtime against wakeup and adjust as necessary */
     if (dryrun || wakeup > runtime) {
-        caldav_alarm_bump_nextcheck(data, wakeup, runtime, NULL);
+        caldav_alarm_bump_nextcheck(data, wakeup, 0, NULL);
```

Comment:
For dryrun you need the zero here (don't change it) but for `wakeup greater` don't we actually need to track the runtime?

Reply:
I don't think we want to adjust the last_run with runtime here because we actually haven't tried to unsnooze.  We got to this point earlier than the actual wakeup time.